discord_bot/discord_bot.py

```diff
@@ -20,13 +21,20 @@ def __init__(self):
         self.gateway_ws_url = None
         self.heartbeat_interval_ms = None
         self.last_seq = None
+        self.logger = None
         self.event_loop = None
         self.websocket = None
         self.session_id = None
 
-    # TODO: Docstrings for setup and run
+    # TODO: Docstrings for setup and runs
     def setup(self):
         self.config = self.load_config(CONFIG_FILE_PATH)
+
+        logging.basicConfig(format='%(asctime)s %(module)s %(levelname)s: %(message)s',
```

Comment:
We can keep this formatting if you like the way it looks, but it's optional. We can just calll `logging.basicConfig()` with no arguments and it will work.